### PR TITLE
Style tweaks

### PR DESCRIPTION
**Pull request by aoswalt**
Fix field squish, adjust character sizes, and a few small tweaks to various elements.

### CODE REVIEW

File: styles/gauntlet.css [hunk 1]
```diff
@@ -84,7 +106,6 @@ COMBAT LAYOUT STYLES
 }
 
 #field {
-	height: 250px;
 	width: 90%;
 }
```
Comment (josephkane) [inline styles/gauntlet.css]: Is there any way we could make this height responsive? Its way tall on my screen and throws off all the formatting.

File: styles/gauntlet.css [hunk 2]
```diff
@@ -100,9 +121,17 @@ COMBAT LAYOUT STYLES
...
 	width: 100%;
 	font-size: 1.3em;
+	font-weight: bolder;
+	background-color: firebrick;
+	color: white;
+}
+
+#attack-button:hover {
+	background-color: red;
 }
```
Comment (josephkane) [inline styles/gauntlet.css:131]: Might be cool to have this match the color of the mushrooms (#ea0f38)?

File: styles/gauntlet.css [hunk 3]
```diff
@@ -63,6 +69,22 @@ SELECT LAYOUT STYLES
 	margin-bottom: 20px;
 }
 
+#species-select[disabled],
+#class-select[disabled],
+#weapon-select[disabled],
+input[disabled] {
+	background-color: lightgrey;
+}
```
Comment (josephkane) [inline styles/gauntlet.css:77]: For some reason on my computer, when these are disabled they are bigger than they are when the are enabled, and they throw off the formatting.

Comment (aoswalt) [inline styles/gauntlet.css:77]: I can't replicate this, and the background color shouldn't affect it. Once everything is merged, I will see if it happens and go from there.

**Comment by josephkane** [comment]
👍

@aoswalt  I'm all merged up, let me know once you are too. Last thing I'll do is mess with the font and comment display if thats cool with you?

**Pull request merged**